modules/lightning_telemetry/src/Event.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Drupal\lightning_telemetry;
+
+/**
+ * AmplitudeEvent.
```

Comment:
This doc comment seems kinda pointless?

Reply:
It's just required for coding standards.